templates/minimal/runtime/src/lib.rs

```diff
@@ -36,6 +36,60 @@ use polkadot_sdk::{
 	*,
 };
 
+/// Provides getters for genesis configuration presets.
+pub mod genesis_config_presets {
+	use crate::{
+		interface::{AccountId, Balance, MinimumBalance},
+		sp_genesis_builder::PresetId,
+		sp_keyring::AccountKeyring,
+		BalancesConfig, RuntimeGenesisConfig, SudoConfig,
+	};
+
+	use alloc::{vec, vec::Vec};
+	use polkadot_sdk::{sp_core::Get, sp_genesis_builder};
+	use serde_json::Value;
+
+	// Returns a genesis config preset populated with given parameters.
+	fn testnet_genesis(endowed_accounts: Vec<(AccountId, u64)>, root: AccountId) -> Value {
```

Comment:
nit: i'd be tempted to remove this function, and inline it into `development_config_genesis`. Probably will be more readable.

Reply:
Done here: [2a1d44b](https://github.com/paritytech/polkadot-sdk/pull/5868/commits/2a1d44b8b9698de1ac59d38e87438f4bf8a484fc).